aws/table_aws_cloudfront_origin_access_identity.go

```diff
@@ -0,0 +1,164 @@
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Comment", "CloudFrontOriginAccessIdentity.CloudFrontOriginAccessIdentityConfig.Comment"),
+			},
+			{
+				Name:        "etag",
+				Description: "The current version of the origin access identity's information.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCloudFrontOriginAccessIdentity,
+				Transform:   transform.FromField("ETag"),
+			},
+
+			//  Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Id", "CloudFrontOriginAccessIdentity.Id"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCloudFrontOriginAccessIdentityARN,
+				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listCloudFrontOriginAccessIdentities(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("listCloudFrontOriginAccessIdentities")
+
+	// Create session
+	svc, err := CloudFrontService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.ListCloudFrontOriginAccessIdentitiesPages(
+		&cloudfront.ListCloudFrontOriginAccessIdentitiesInput{},
+		func(page *cloudfront.ListCloudFrontOriginAccessIdentitiesOutput, isLast bool) bool {
+			for _, identity := range page.CloudFrontOriginAccessIdentityList.Items {
+				d.StreamListItem(ctx, identity)
+			}
+			return !isLast
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getCloudFrontOriginAccessIdentity(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getCloudFrontOriginAccessIdentity")
+
+	// Create session
+	svc, err := CloudFrontService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	var identityID string
+	if h.Item != nil {
+		identityID = *h.Item.(*cloudfront.OriginAccessIdentitySummary).Id
+	} else {
+		identityID = d.KeyColumnQuals["id"].GetStringValue()
+	}
+
+	params := &cloudfront.GetCloudFrontOriginAccessIdentityInput{
+		Id: aws.String(identityID),
+	}
+
+	op, err := svc.GetCloudFrontOriginAccessIdentity(params)
+	if err != nil {
+		return nil, err
+	}
+
+	return op, nil
+}
+
+func getCloudFrontOriginAccessIdentityARN(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getCloudFrontOriginAccessIdentityARN")
+	originAccessIdentityData := *originAccessIdentityID(h.Item)
+
+	c, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+
+	commonColumnData := c.(*awsCommonColumnData)
+	arn := "arn:" + commonColumnData.Partition + ":cloudfront:" + commonColumnData.AccountId + ":originaccessidentity/" + originAccessIdentityData
```

Comment:
arn format should be `arn:${Partition}:cloudfront::${Account}:origin-access-identity/${Id}`